templates/docs/examples/base/code-block.html

```diff
@@ -4,9 +4,9 @@
 {% block standalone_css %}base{% endblock %}
 
 {% block content %}
-<pre>
-<code>function() {
+<div class="p-code-snippet">
```

Comment:
This example should stay as it is, because it's example of base styles, not snippet component.